action.yml

```diff
@@ -21,9 +21,14 @@ inputs:
   password:
     description: 'Password to use as credential to authenticate to the registry'
     required: true
+  tls-verify:
+    description: 'Require HTTPS and verify certificates when contacting registry'
```

Comment:
can you update this to match the readme description

`Verify TLS certificates when contacting the registry.`